Makefile

```diff
@@ -101,6 +101,9 @@ test: jsonnet libjsonnet.so libjsonnet_test_snippet libjsonnet_test_file
 	cd test_suite ; ./run_tests.sh
 	cd test_suite ; ./run_fmt_tests.sh
 
+reformat:
```

Comment:
I'm worried that reformatting on every build (== adding to `all`) may be annoying. 

:+1: for `.PHONY` I forgot about it. 

Also I guess I'll add a separate `test-formatting` target for use in TravisCI etc.